packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js

```diff
@@ -12,19 +20,27 @@ export default class IpByoipAggregateController {
     this.Alerter = Alerter;
     this.$translate = $translate;
     this.atInternet = atInternet;
+    this.byoipUsageGuideUrl =
+      BYOIP_USAGE_GUIDE_URL[coreConfig.getUser().ovhSubsidiary] ||
+      BYOIP_USAGE_GUIDE_URL.DEFAULT;
   }
 
   $onInit() {
+    this.showErrorMessage = false;
+    this.showSlicingNotAvailableError = false;
     this.atInternet.trackPage({ name: TRACKING_PREFIX_AGGREGATE });
     this.IpByoipService.getAvailableAggregationConfigurations(this.ip.ipBlock)
       .then(({ data }) => {
         this.aggregationIps = data;
         if (this.aggregationIps.length) {
           [this.selectedAggregationIp] = this.aggregationIps;
+        } else {
+          this.showErrorMessage = true;
+          this.showSlicingNotAvailableError = true;
```

Comment:
**nitpick:** Pretty confusing with both the variables. Can we try. and simplify the implementation with single variable.

Reply:
Remove it now.